pkg/constant/env.go

```diff
@@ -30,17 +30,19 @@ const (
 
 const (
 	// Lorry env names
-	KBEnvClusterName     = "KB_CLUSTER_NAME"
-	KBEnvComponentName   = "KB_COMP_NAME"
-	KBEnvClusterCompName = "KB_CLUSTER_COMP_NAME"
-	KBEnvWorkloadType    = "KB_WORKLOAD_TYPE"
-	KBEnvCharacterType   = "KB_SERVICE_CHARACTER_TYPE"
-	KBEnvServiceUser     = "KB_SERVICE_USER"
-	KBEnvServicePassword = "KB_SERVICE_PASSWORD"
-	KBEnvServiceRoles    = "KB_SERVICE_ROLES"
-	KBEnvServicePort     = "KB_SERVICE_PORT"
-	KBEnvDataPath        = "KB_DATA_PATH"
-	KBEnvTTL             = "KB_TTL"
-	KBEnvMaxLag          = "KB_MAX_LAG"
-	KBEnvEnableHA        = "KB_ENABLE_HA"
+	KBEnvClusterName            = "KB_CLUSTER_NAME"
+	KBEnvComponentName          = "KB_COMP_NAME"
+	KBEnvClusterCompName        = "KB_CLUSTER_COMP_NAME"
+	KBEnvWorkloadType           = "KB_WORKLOAD_TYPE"
+	KBEnvCharacterType          = "KB_SERVICE_CHARACTER_TYPE"
+	KBEnvServiceUser            = "KB_SERVICE_USER"
+	KBEnvServicePassword        = "KB_SERVICE_PASSWORD"
+	KBEnvServiceRoles           = "KB_SERVICE_ROLES"
+	KBEnvServicePort            = "KB_SERVICE_PORT"
+	KBEnvDataPath               = "KB_DATA_PATH"
+	KBEnvTTL                    = "KB_TTL"
```

Comment:
pls add some comments for these envs. hard to infer the purpose from their names.